plugins/extractors/clickhouse/clickhouse.go

```diff
@@ -37,12 +46,15 @@ func (e *Extractor) Extract(ctx context.Context, configMap map[string]interface{
 	if err != nil {
 		return
 	}
-	result, _ := e.getTables()
-	out <- result
+	err = e.getTables()
```

Comment:
let's rename this function to `extractTables`.
`getTables` implies it will return a list of tables. wdyt?

Reply:
Yes, makes more sense